js/src/components/paid-ads/campaign-assets-form.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Internal dependencies
+ */
+import AdaptiveForm from '.~/components/adaptive-form';
+import validateCampaign from '.~/components/paid-ads/validateCampaign';
+
+/**
+ * @typedef {import('.~/components/types.js').CampaignFormValues} CampaignFormValues
+ */
+
+/**
+ * Renders a form based on AdaptiveForm for managing campaign and assets.
+ *
+ * @augments AdaptiveForm
```

Comment:
💅  Not blocker. Even being a sinonym, I would use a more widely used keyword for this `@extends`

Reply:
The reason for using `@augments` instead of `@extends` was to convey that the props of `CampaignAssetsForm` are inherited from `AdaptiveForm` and that they are not language-level inheritance. Considering `@augments` isn't quite the right tag, it might be better to remove it and add a code comment instead.